### ISSUE TITLE
General discussion

### ISSUE THREAD

**Issue by SOF3** [SOF3/forums-common-sense#4]
This issue exists to allow uncategorized discussion of the wiki, without the cost of starting another issue.

**Comment by CaitiffMCPE** [comment]
Do not worry about the cost of starting another issue, I have already started it by jumping in the discussion like a person who likes to get in arguments very much :) I am Caitiff. 

You do not have to create the common-sense for everyone. The only way for some people to create common-sense for themselves is to do it by themself. This won't happen if people do not have desire because if people had had it they would have already visited other, more detailed sources of common-sense by now. However, I respect that you want do something for everyone but it is just inevitable. This is like creating an entire library that already exists. Go ahead with your idea, I will still support it. Who knows, maybe this will suprise us.

**Comment by CaitiffMCPE** [comment]
It is just like the general discussion about the new PocketMine name. It is impossible to get everyone to agree with an idea. If you let other people decide your future then you will pretty much be their prisoner forever. I am not saying people shouldn't make general decisions to collect opinions but it is just that you can not get everyone to agree with your idea... and that is pretty much it.  You just shouldn't think of what everyone says. That's why we say quality not quantity and that is why the founders should choose the name that suits "PocketMine" the best.

**Comment by SOF3** [comment]
@CaitiffMCPE Actually, my concept of common sense is to create a standard, or a set of less strict forum rules/etiquette that the majority of the community agrees to follow, so as to prevent any unnecessary argument or misunderstanding.

Your second comment very well can be a great elaboration about the disadvantages of populism or cyber bullying. But how is it related to this wiki? This wiki is a product of community that aims for integration. We recommend following this wiki, but we certainly don't force anyone to follow them. This wiki is particularly useful when newcomers do not understand some of the common abbreviations used in this section of the web.

This wiki can serve to prevent the community's fragmentation due to generation gap (sounds hilarious, but this is true). While the time keeps on going in the long term, this wiki both helps newcomers to integrate into our community _as well as_ help the elder members understand how the conventions are being changed. This is for mutual integration.

On the other hand, this wiki is actually about the **values** that most of us agree to have, something that already existed before being written out. This is why a "Controversy" label exists. If there is any controversy, we aren't trying to forcefully integrate them together and settle the problem, much less by any authorities or any referendums or fights. Just like we aren't going to put "Everyone should write 'Amen' before ending every post" in this wiki. (Disclaimer: I don't have any religions)

**Comment by CaitiffMCPE** [comment]
'Traci'. wtf? but I did not

**Comment by SOF3** [comment]
@CaitiffMCPE you know you can still edit it.

**Comment by CaitiffMCPE** [comment]
for some reason I have big problems with understanding pointers and references, i just can't get it

**Comment by CaitiffMCPE** [comment]
done

**Comment by SOF3** [comment]
@CaitiffMCPE general discussion doesn't mean that you can post any irrelevant comments here. Please post comments related to this wiki.

**Comment by CaitiffMCPE** [comment]
That comment is related to the wiki

**Comment by CaitiffMCPE** [comment]
Wait, I am not banned from editing the wiki, right? (because of the irrelevant comment)

**Comment by SOF3** [comment]
@CaitiffMCPE If you are, you can't even comment now. Now please stop posting irrelevant comments in this issue. This issue is for discussion about the wiki, not asking whether you are banned.

**Comment by CaitiffMCPE** [comment]
Lmao I am such a troll hahahah

**Comment by CaitiffMCPE** [comment]
It looks like you were talking to yourself this whole time! haha

**Comment by CaitiffMCPE** [comment]
What? Being a dev is not related to ****

**Comment by CaitiffMCPE** [comment]
Amen

**Comment by CaitiffMCPE** [comment]
there are quite a lot of abbrs to add for example OS, CPU, RAM, PC, etc

**Comment by CaitiffMCPE** [comment]
I wonder how do you know so much about everything. Perhaps it is your love to the knowleadge. I wish I had the same motivation :/

**Comment by CaitiffMCPE** [comment]
Well, not everything, but you seem to know a lot

**Comment by CaitiffMCPE** [comment]
It is cool that people start to think like you do. Nice work there

**Comment by CaitiffMCPE** [comment]
It is like an extreme illumination

**Comment by SOF3** [comment]
@CaitiffMCPE Actually I think @ifvictr overdid it. We don't need abbreviations for everything, only ones that are specific to the forums and probably cause confusion. PC, OS, CPU, RAM, VPN, VPS,,etc., are all so obvious.
I originally only added FTP because of its conflict with FloatingTextParticle. Other from conflicts like this, I think that all abbreviations that you can easily find the apparent meaning from Wikipedia should be removed.

**Comment by ifvictr** [comment]
@SOF3 I don't think I overdid it, it may be obvious to us, but not to some people. We cannot expect everyone to have the same amount of experience as us.

**Comment by SOF3** [comment]
@ifvictr but those thing are globally consistent. For example, if someone doesn't know what VPS mean, reading this wiki won't help them a lot. This wiki isn't a wiki for knowledge, but a wiki for reaching consensus, after all.

**Comment by CaitiffMCPE** [comment]
You know, FTP can be confused with File Transfer Protocol, right?

**Comment by TheDiamondYT1** [comment]
Mention how JSON can be minified ane loaded quicker than YAML or unminified JSON.
 
---

I'm sure there's a specific name for minified JSON but i forgot it

**Comment by SOF3** [comment]
@TheDiamondYT1 note that it is pointless to minimize small files. For example, on Windows, a file is saved in blocks of 4096 B iirc, i.e. if the file is smaller than 4096 B, it will still occupy 4096 B and reading that file will still read 4096 B.

**Comment by TheDiamondYT1** [comment]
I read somewhere I think wikipedia it's more efficient to load JSON when it's minified but they had a special name for it

**Comment by SOF3** [comment]
I don't think this makes sense. The cost of reading useful content from the filesystem would always be way greater than those unnecessary bytes.
